_indexes/2018/contributors.md

```diff
@@ -0,0 +1,1335 @@
+---
+layout: page
+name: Contributors
+volume: '2018'
+title: 'Index 2018: Contributors'
+wrapperclass: 'index-contributors'
+toc: 2018 Contributors
+---
+
+3dwardsharp  32–33, 52–53
```

Comment:
😼

Reply:
i say defer until later personally, let me know if you, @edwardsharp, have strong feelings on this matter - also - what do you think of the styling (or lack thereof)? can't remember what it was like before